docker/iml-action-runner.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.39 as builder
+WORKDIR /build
+COPY . .
+RUN cd iml-services/iml-action-runner && cargo build --release
+
+FROM rust:1.39
+COPY --from=builder /build/target/release/iml-action-runner /usr/local/bin
+RUN apt-get update \
```

Comment:
I don't think we use postgres in this service.

Reply:
I removed the postgres client, rebuilt, and it now complains without it.
```
iml-action-runner_1  | /usr/local/bin/wait-for-dependencies.sh: line 35: psql: command not found
```

Reply:
Since all three of these new services complain about the postgres client after taking it out I looked to see what was common between them that might be using it. I believe its `iml-wire-types`: https://github.com/whamcloud/integrated-manager-for-lustre/blob/83ab452b16f6a39420e812cde1efa4cff0e6f274/iml-wire-types/Cargo.toml. I bet it's the tokio-postgres dep.

Reply:
No... it's this:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/a58221b4dbdb0c8837a7c2dd2d17cd2bcdecfcd7/docker/wait-for-dependencies.sh#L35

We should try and gate that check to only services where it's needed.

Reply:
perhaps a different entrypoint for containers needing postgres?